packages/@webex/plugin-meetings/src/meeting/request.ts

```diff
@@ -792,4 +792,25 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       }
     });
   }
+
+  /**
+   * Make a network request to enable or disable reactions.
+   * @param {boolean} options.enable - determines if we need to enable or disable.
+   * @param {locusUrl} options.locusUrl
+   * @returns {Promise}
```

Comment:
Please do not define the the parameter types in JSDoc in TS files, they already defined in the function definition (line 802)

```suggestion
   * @param options toggle reaction options
   * @returns service Response
```

Reply:
currently we are using jsDoc and i has a ticket to move to tsdoc , With the current eslint rules its going to throw and error

So i would keep it as it is for now unless we move